src/utils/parseJson.ts

```diff
@@ -3,12 +3,40 @@
  *  Licensed under the MIT License. See License.md in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+import * as jsonc from 'jsonc-parser';
+import { localize } from '../localize';
+
 /**
- * Has extra logic to remove a BOM character if it exists
+ * Has extra logic to remove a BOM character if it exists and handle comments
  */
 export function parseJson<T extends object>(data: string): T {
     if (data.charCodeAt(0) === 0xFEFF) {
         data = data.slice(1);
     }
-    return <T>JSON.parse(data);
+
+    const errors: jsonc.ParseError[] = [];
+    const result: T = <T>jsonc.parse(data, errors, { allowTrailingComma: true });
+    if (errors.length > 0) {
+        const [line, column]: [number, number] = getLineAndColumnFromOffset(data, errors[0].offset);
+        throw new Error(localize('jsonParseError', '{0} near line "{1}", column "{2}"', jsonc.printParseErrorCode(errors[0].error), line, column));
```

Comment:
Are there ever multiple errors?  Would we ever want to output all of the errors and direct them to the console?

Reply:
Yeah there can be multiple errors, but displaying only the first error is consistent with the previous behavior. Plus, VS Code will handle displaying all errors in the "Problems" tab when you open up the file, like this:
<img width="360" alt="Screen Shot 2020-10-05 at 3 40 12 PM" src="https://user-images.githubusercontent.com/11282622/95139367-2fab4200-0721-11eb-8e04-0b5930f39c96.png">